godot/src/test/animated_material/animated_material.gdshader

```diff
@@ -0,0 +1,32 @@
+// Shader by Polysthetic (https://youtu.be/-c38gFbGO6U)
```

Comment:
what does this material (and the scene) do? can you put some description?

Reply:
That shader was added without intention.
I renamed the scene to `godot_animated_material`, so basically, it's using a MeshInstance3D with a ShaderMaterial and its texture is being animated with a sprite sheet.

It's more an example than a test... should I remove it? :thinking: